apps.yml

```diff
@@ -4248,3 +4248,16 @@ apps:
     url: https://auth.mozilla.auth0.com/samlp/VezzSFjr5YxbFicOqhLmnL8UfUGALQtl
     vanity_url:
     - /notion
+- application:
+    authorized_groups:
+    - team_moco
```

Comment:
nonblocking review: should this be `team_pocket`?

Reply:
Ah good catch, I'll add.

Reply:
`team_pocket` is a manually curated subset of `team_moco`.  It's "folks who are pocket" and we have never been able to determine who those folks are from workday info.
If you have `team_moco` you don't need `team_pocket`.  If you need more than `team_pocket` then stay with `team_moco`.  This comment is an "I don't know who all needs it" but it's just to call attention to `team_pocket` existing.  "Choose one, your call."  Sorry for not being clear to start from.